bin/oneoff/reorganize_level_files.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+require_relative '../../lib/cdo/only_one'
+abort 'Script already running' unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+
+# We currently have ~31592 un-reorganized level files; we want to batch this
+# update a bit so GitHub doesn't have to deal with 30k files in a single PR.
+#
+# GitHub doesn't like a large number of single files in a single directory
+# (which is why we're doing this work in the first place), but also doesn't
+# like too many files in a single PR. By default, GitHub will only load the
+# first 3000 changed files in a PR, so we want to limit the number of files we
+# reorganize at once to be under that limit so the PRs in which these changes
+# happen are still at least somewhat viewable.
+#
+# We plan to execute this script on the levelbuilder machine and capture the
+# change with the regularly scheduled staging scoop PRs, so we want to leave a
```

Comment:
I'm wondering if it's at least worth making separate commits for the changes from this script vs regular levelbuilder updates. It can be useful to look at the content changes in the levelbuilder scoop from time to time and it might be overwhelming to look over 2500+ files

Reply:
Yeah, that's a good point. I think it would be reasonable for us to plan to do a manual scoop both before and after running this script. Note that it does take a couple of minutes to finish, so it's possible some regular changes could get mixed in; but I think that's an acceptable blemish if it does happen.

So, the new plan would be something like:

```bash
bin/content-push && bin/oneoff/reorganize_level_files.rb && bin/content-push
```

Reply:
Yep that sounds good to me! I also think it's okay if a couple of changes got pulled in